modules/RTC/ScreenObtainer.js

```diff
@@ -76,11 +76,9 @@ const ScreenObtainer = {
         this.options = options = options || {};
         gumFunction = gum;
 
-        if (RTCBrowserType.isFirefox()) {
-            initFirefoxExtensionDetection(options);
-        }
-
-        if (RTCBrowserType.isNWJS()) {
+        if (this.options.disableDesktopSharing) {
```

Comment:
It's probably very subjective, but we only have those lines at the end of the if-else sequence : 
if (!obtainDesktopStream) {
logger.info('Desktop sharing disabled');
}
this.obtainStream = obtainDesktopStream;
so I think it would improve code readability to handle the the disable case right here, log whatever we need to, assign null to obtain stream if needed and return. It's just weird to me to have if () { nothing }.